app/test_main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from unittest.mock import patch
+from app.main import can_access_google_page
+
+
```

Comment:
better to use  @pytest.parametrize

Reply:
Yes, it seems that I did not quite understand the task from the 1st time)